docs/src/details/integration/renderers/eve.rst

```diff
@@ -0,0 +1,233 @@
+*****
+
+Board Parameters
+----------------
+
+Create a ``lv_draw_eve_target_parameters.h`` and add it to the include search
+path of your build.
+
+Here is an example ``lv_draw_eve_target_parameters.h`` for the Riverdi RVT50HQBNWC00-B
+which has a BT817Q --- the EVE chip.
+These parameters were taken from the collection of configs provided by the EVE
+library supporting this renderer.
+`See here <https://github.com/RudolphRiedel/FT800-FT813/blob/ee35fa45c37aed78dc4f1196b874ee1c8f80429f/EVE_config.h#L856-L872>`__.
+Check there for your board parameters.
+
+.. code-block:: c
+
+    #pragma once
+
+    #define EVE_HSIZE (800L)
+    #define EVE_VSIZE (480L)
+    #define EVE_VSYNC0 (0L)
+    #define EVE_VSYNC1 (4L)
+    #define EVE_VOFFSET (8L)
+    #define EVE_VCYCLE (496L)
+    #define EVE_HSYNC0 (0L)
+    #define EVE_HSYNC1 (4L)
+    #define EVE_HOFFSET (8L)
+    #define EVE_HCYCLE (816L)
+    #define EVE_PCLK (3L)
+    #define EVE_PCLKPOL (1L)
+    #define EVE_SWIZZLE (0L)
+    #define EVE_CSPREAD (0L)
+    #define EVE_HAS_CRYSTAL
+    /* notice EVE_HAS_GT911 is not defined */
+    #define EVE_GEN 4
+
+Add it to the include search path of your build by adding the compiler
+flag ``-I /path/to/dir/that/has/the/file/``. In CMake, see
+``include_directories`` and ``target_include_directories``. In many cases,
+placing the file in the same directory as your ``lv_conf.h`` will work.
+
+User-Provided SPI HAL Implementation
+------------------------------------
+
+The user is required to implement four functions which the LVGL EVE
+renderer will call to perform SPI communication with the EVE chip.
+The functions can simply be defined in your own files. They must not be
+``static``.
+
+.. code-block:: c
+
+    #include "lvgl/lvgl.h"
+
+    void lv_draw_eve_target_powerdown(bool powerdown)
+    {
+        ...
+    }
+
+    void lv_draw_eve_target_spi_cs(bool selected)
+    {
+        ...
+    }
+
+    void lv_draw_eve_target_spi_transmit(uint8_t * src, uint32_t len)
+    {
+        ...
+    }
+
+    void lv_draw_eve_target_spi_receive(uint8_t * dst, uint32_t len)
+    {
+        ...
+    }
+
+The ``powerdown`` parameter of ``lv_draw_eve_target_powerdown`` when ``true``
+means that the ``PD_N`` pin should be set low (power off)
+and ``false`` means that it should be set high (power on).
+
+The ``selected`` parameter of ``lv_draw_eve_target_spi_cs`` when ``true``
+means that the ``CS_N`` pin should be set low (chip is selected)
+and ``false`` means that it should be set high (chip is deselected).
+
+The ``src`` parameter of ``lv_draw_eve_target_spi_transmit`` is not a
+pointer to ``const`` since your implementation may naturally want to
+overwrite the outgoing data with (meaningless) incoming data
+(which will be ignored by LVGL).
+
+In practice, the ``len`` parameters will be only 1 or 4 for transmit, and
+always 1 for receive. A forward-compatible implementation should account
+for other values, though.
+
+Here is an example implementation for ESP32-S3.
```

Comment:
Can you replace these descriptions by adding dummy function calls in the skeletons above?